slides/03-how-we-work/1070-expectations.md

```diff
@@ -5,5 +5,5 @@
 - Collaborating with others to complete stories
 - Reach out to other teams, identify opportunities to share best practice and re-use components​
 - Publicise libraries and services​
-- Up to 20% of time on non-project work​
-- Participate in away days​
\ No newline at end of file
+- Aspiration for up to 20% of time on non-project work​
```

Comment:
:(

Reply:
Yeah I know, it is something we need to get to, but we got a lot of progress to make with portfolios buying into this in the new model, and I don't want to make promises we're not yet able to keep